op.c

```diff
@@ -384,8 +384,13 @@ val_print(struct val val) {
   case VT_RAW: {
     u32 len;
     u8 *data = byte_pool_get(val.ind, &len);
-    printf("0x");
-    print_bytes(data, len, 0);
+    if (len == 0) {
+        printf("[]");
+    }
+    else {
+        printf("0x");
```

Comment:
if `[]` means empty string than we should switch `0xdeadbeef` to `[deadbeff]`

Reply:
I'll roll it back.